nupic/research/frameworks/vernon/mixins/centroid_figure_1b.py

```diff
@@ -0,0 +1,111 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
Can you move this to dendrites/mixins? I don't think such a specific file should be in the Vernon directory structure.

Reply:
Done.

Reply:
Looks like the init files and imports were changed, but the file itself wasn't moved in the commit.

Reply:
Are you sure? It tells me the file was indeed moved. Also,

- [dendrites/mixins/](https://github.com/karangrewal/nupic.research/tree/vernon-dendrites-3/nupic/research/frameworks/dendrites/mixins) folder on this branch,
- [vernon/mixins folder](https://github.com/karangrewal/nupic.research/tree/vernon-dendrites-3/nupic/research/frameworks/vernon/mixins) on this branch

and only the latter contains the file `centroid_figure_1b.py`.

Reply:
Yes, you're right. I think my various github tabs did not update in sync, even after a refresh. Sorry about that.